client/js/screens/DailyReports/DailyReports.js

```diff
@@ -10,6 +10,7 @@ class DailyReports extends Component {
       test: this.props.dailyReportsQuey
```

Comment:
change your state in the component did mount. We should never assign state value from props in the constructor.